packages/cashier/src/components/cashier-search-box/cashier-search-box.tsx

```diff
@@ -35,7 +50,12 @@ const CashierSearchBox = ({ className, onClear, onSearch, placeholder, search_te
                 {({ submitForm, values: { search }, setFieldValue }) => (
                     <Form>
                         <FormField name='search'>
-                            {({ field }) => (
+                            {(field: {
+                                onChange: (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+                                onBlur: () => void;
+                                name: string;
+                                value: string;
+                            }) => (
```

Comment:
Typed it the way it's presumably supposed to be typed https://formik.org/docs/api/field#rendering